### DISCUSSION TITLE
Mirth Http Listener with multiple destinations

### DISCUSSION THREAD

**Discussion by manojkvn** [nextgenhealthcare/connect#6301] [Q&A]
hi Mirth Experts,

we have written http listener as source connector that has destination 1 and destination 2(will wait till destination 1 competes), the response for the source connector is destination 1, in this case can some one confirm if response is recieved upoun completion of destination 2 (ie., all destinations are completed) or is completion of destination 1 is enough. 

help is appreciated

**Comment by pacmano1** [comment]
I would think you can test this yourself, no?    Set a sleep in destination 2 somewhere like a transformer.

`java.lang.Thread.sleep(10000) // sleep 10 seconds;`
